_posts/2024-04-01-kubecon-2024.md

```diff
@@ -0,0 +1,244 @@
+
+As promised in their title, they provided five simple security mechanisms to prevent these attacks, along with practical insights and tool recommendations.
+
+1.	**Control Plane Security:** Ensure proper management of the control plane basic, which are mostly covered with managed K8s. Enforce authentication on the API server and minimize its network exposure.
+2.	**Block Cloud Metadata Service Access:** Prevent workloads from accessing the cloud metadata service. While GKE Workload Identity provides enough protection, AWS IAM roles for Service Accounts & Azure Workload Identity do not. Blocking IMDS access using a network policy is required for these platforms.
+3.	**Understand Workload Privileges:** Be aware of the access levels and permissions associated with your workload.
+4.	**Intentional Cluster Management:** Be deliberate about what runs in your cluster. Avoid creating unnecessary resources that may be forgotten or pose security risks.
+5.	**Application Security:** Utilize modern frameworks and regularly update libraries, frameworks, and dependencies.
+
+In conclusion, the talk not only provided valuable insights into the complexities of securing cloud-native environments but also offered practical guidance for addressing these challenges effectively.
+
+## [Product Market Misfit: Adventures in User Empathy - Mitch Connors, Aviatrix](https://www.youtube.com/watch?v=jpIWERlrXT4){:target="_blank" rel="noopener noreferrer"}
+
+Far too often, software developers invest considerable amount of time and effort into building features or products, only to discover they are either disliked or unused by their target audience - a scenario we've encountered firsthand in our day-to-day work.
+
+In his talk, Mitch Connors delves into this issue and provides valuable insights on how to tackle it.
+
+Developers often make the mistake of building what they believe their users need, rather than what they actually require.
+This misalignment originates from differing perspectives of developers and users.
+Connors effectively illustrates this through the use of some real-world examples.
+
+His key advice for tackling this problem is to foster empathy for your users:
+
+1. Get into their shoes; experience what they experience. By doing so, a shared perspective can be developed.
+2. Maintain an open mindset and never stop listening. Embrace criticism, especially if you truly love what you've built.
+
+While Connors' talk certainly differed from all others we attended, it was very interesting and engaging.
+We strongly believe every software developer, as well as anyone involved in creating products for groups of users, could greatly benefit from the insights he provides.
+
+## [Keptn: a deep dive](https://youtu.be/eoK3BQQlsIE?si=0plqlHEv1hX4YCu7){:target="_blank" rel="noopener noreferrer"}
```

Comment:
@DuncanCasteleyn Maybe try to focus more on what Keptn fixes instead of "how it does it".